### PR TITLE
Jest test

### PR DESCRIPTION
**Pull request by Addisu87**
We are testing all the operations, calculations, and components of the app using the jest and react-testing library.

### CODE REVIEW

File: src/test/__snapshots__/App.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+exports[`App renders correctly 1`] = `
+<div
+  className="App"
+>
+  <div
+    className="sc-bczRLJ gUDtTg"
+  >
+    <h2
+      className="sc-gsnTZi dayyfU"
+    >
```
Comment (BenMukebo) [inline src/test/__snapshots__/App.test.js.snap:12]: - [ ] Excellent use of snapshot tests 👏🏻 , but according to the project's requirements, all components must be tested; I only see "App.tet.js.snap," "Button.test.js.snap," and "Navbar.test.js.snap." Please add Jest snapshots for other components, such as (calculator.js, home.js, and quotes.js). 👨🏻‍🎨

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Addisu87, @Olamarx,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
-  All the functionalities & scenarios of the `operate.js` and `calculate.js` modules has been covered using Jest.✔️ 
- You have used React Testing Library to simulate user interaction.✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



- [ ] Kindly, Add a descriptive README file to your project, it will be great to add instructions on how to run the tests. This includes `npm install`  `npm run test`.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/__snapshots__/App.test.js.snap (hunk 1 above)
Comment (Olamarx) [reply src/test/__snapshots__/App.test.js.snap:12]: - [ ] Hello @BenMukebo, thank you for the review, we actually tested other components such as the home page, and the calculator page via the app page. Attached to this comment is a phone screenshot of the test. Kindly check, please.

Screenshot_20220707-112619_Chrome

**Review by Emmyn5600 — APPROVED** [review]
Hi @Addisu87 and @BenMukebo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**